torch/cuda/amp/grad_scaler.py

```diff
@@ -355,11 +355,13 @@ def step(self, optimizer, *args, **kwargs):
                 )
                 optimizer.grad_scale = scaler
                 optimizer.found_inf = found_inf
+                optimizer.grads_unscaled = optimizer_state["stage"] == OptState.UNSCALED
```

Comment:
Is there any case where we want to pass a non-None `grad_scale` and `grads_unscale == True` ? If so, is the new flag actually redundant and we could just modify above:
```python
optimizer.grad_scale = scaler if optimizer_state["stage"] == OptState.UNSCALED else None
```

Reply:
oh it looks right. thank you for the swift response